src/components/layouts/Notification.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from 'react';
+import { useAuth } from '../../context/AuthProvider';
+import Image from 'next/image';
+import assets from '../../variables/images';
+import styles from './Notification.module.css';
+import { fetchNotifications, markAsRead } from '../../service/api/notification';
+
+const Notification = () => {
+  const [notifications, setNotifications] = useState([]);
+  const [showNotifications, setShowNotifications] = useState(false);
+  const { user } = useAuth();
+
+  const getNotifications = async () => {
+    if (!user) return;
+    try {
+      const response = await fetchNotifications(user.id, true); // includeRead를 true로 설정
+      setNotifications(response);
+    } catch (error) {
+      console.error('알림을 가져오는 중 오류가 발생했습니다:', error);
+    }
+  };
```

Comment:
```suggestion
const getNotifications = async () => {
  if (!user) return;
  setLoading(true); 
  try {
    const response = await fetchNotifications(user.id, true);
    setNotifications(response);
    setError(null);
  } catch (error) {
    setError('알림을 가져오는 중 문제가 발생했습니다. 다시 시도해주세요.'); 
  } finally {
    setLoading(false);
  }
};
```

이전 코드는 에러 발생 시 콘솔에만 에러 메시지를 출력하고, 사용자에게는 아무런 피드백을 제공하지 않고 있습니다.
이렇게 수정해보시는 건 어떨까요?